Getting-Started/Setup/Server-Setup/Load-Balancing/azure-web-apps.md

```diff
@@ -0,0 +1,58 @@
+---
+versionFrom: 8.0.0
+---
+
+## Load Balancing Azure Web Apps
+
+Ensure you read the [overview](index.md) before you begin - you will need to ensure that your ASP.NET & logging configurations are correct.
+
+### Azure Requirements
+
+* You will need to setup 2 x Azure Web Apps - one for the master (administrative) environment and another for your front-end environment
+* You will need 1 x SQL server that is shared with these 2 web apps
+
+### Lucene/Examine configuration
+
+The single instance backoffice Web App should be set to use [SyncTempEnvDirectoryFactory](file-system-replication.md#examine-directory-factory-options).
+
+The multi instance front end Web App should be set to use [TempEnvDirectoryFactory](file-system-replication.md#examine-directory-factory-options).
+
+### Umbraco TEMP files
+
+Store the Umbraco temporary files in the local server's 'temp' folder. Achieve this by changing this configuration setting to 'true' in the web.config. The downside is that if you need to view this configuration file you'll have to find it in the temp files. Locating the file this way isn't always clear.
```

Comment:
I'm not sure everybody knows which Temp files you are referring to here, but that maybe obvious in the context of where this page sits... but possibly could we say something like

When an instance of Umbraco runs it generates some 'temporary' files on disk... in a non-cloud environment these would be created within the folders of the Web Application...however in an Azure Web App we want these to be created in the local storage of the actual server that Azure happens to be using for the Web App and so  we set this configuration setting to 'true', and the temporary files will be located in the .net temporary folder. This is great for 'speed' of access for Umbraco operation, however, the downside is it's very difficult on Azure to manually browse to this temporary location if you are troubleshooting for any reason.

(Also 'if you need to view this configuration file' ??? with nucache we no longer have an umbraco.config? published cache so is it wrong to refer to it as a configuration file??)